staging/src/k8s.io/api/flowcontrol/v1alpha1/types.go

```diff
@@ -367,14 +375,50 @@ type QueuingConfiguration struct {
 	// +optional
 	AssuredConcurrencyShares int32 `json:"assuredConcurrencyShares" protobuf:"varint,1,opt,name=assuredConcurrencyShares"`
```

Comment:
if the future-us factor a new rate-based limit type, we will rename `assuredConcurrencyShares` to `assuredShares` nest it into a union structure, right?